### ISSUE TITLE
idmmobil cars lane change into ego car

### ISSUE THREAD

**Issue by cho3** [zsunberg/Multilane.jl#11]
Example:

```
nb_lanes = 4
pp = PhysicalParam(nb_lanes,lane_length=100.) #2.=>col_length=8
_discount = 1.
nb_cars=10

rmodel = NoCrashRewardModel()

dmodel = NoCrashIDMMOBILModel(nb_cars, pp)

mdp = NoCrashMDP(dmodel, rmodel, _discount);

s = Multilane.MLState(false,[Multilane.CarState(50.0,4.0,28.125,0.0,Nullable{Multilane.BehaviorModel}()),Multilane.CarState(45.63831155736089,3.0,29.959029299309613,0.6666666666666666,Nullable(Multilane.IDMMOBILBehavior(Multilane.IDMParam(1.0,1.0,2.0,31.0,4.0,4.0),Multilane.MOBILParam(0.5,4.0,0.2),1.0,4))),Multilane.CarState(7.169470572976412,1.0,30.675116224204793,0.0,Nullable(Multilane.IDMMOBILBehavior(Multilane.IDMParam(1.5,1.5,1.4,35.0,4.0,4.0),Multilane.MOBILParam(0.25,4.0,0.2),1.0,8))),Multilane.CarState(98.01576224720353,4.0,27.245023469032592,0.0,Nullable(Multilane.IDMMOBILBehavior(Multilane.IDMParam(1.5,1.5,1.4,27.5,4.0,4.0),Multilane.MOBILParam(0.25,4.0,0.2),1.0,2))),Multilane.CarState(0.42803785502447056,3.0,27.924225639837424,0.6666666666666666,Nullable(Multilane.IDMMOBILBehavior(Multilane.IDMParam(1.5,1.5,1.4,31.0,4.0,4.0),Multilane.MOBILParam(0.25,4.0,0.2),1.0,5))),Multilane.CarState(100.0,2.0,28.125,0.0,Nullable(Multilane.IDMMOBILBehavior(Multilane.IDMParam(1.5,1.5,1.4,27.5,4.0,4.0),Multilane.MOBILParam(0.25,4.0,0.2),1.0,2)))])

```

**Comment by cho3** [comment]
I think it has to do with is_crash not being aware of the consistency checking logic in lane changing (assumes dynamics without consistency checking)

**Comment by cho3** [comment]
Merged with #6 --this is an issue with crashing
